integration_tests/snaps/simple-svn/snapcraft.yaml

```diff
@@ -0,0 +1,11 @@
+name: test-package
+version: 0.1
+summary: svn integration test
+description: use svn
+confinement: strict
+
+parts:
+    svn:
+        plugin: make
+        source: .
+        source-type: svn
```

Comment:
@chadmiller If you are talking about kyrofa's issue, I found that out already, but I forgot to update my code. My bad, but it's fixed now.

What else are you referring to if not that? Can you clarify?